src/test/java/org/testobject/appium/testng/RemoteWebDriverCalculatorWatcherTestTestNG.java

```diff
@@ -26,42 +26,39 @@ public void beforeTest() throws MalformedURLException {
 
 		capabilities.setCapability("testobject_api_key", "YOUR_API_KEY");
 		capabilities.setCapability("testobject_app_id", "1");
-		capabilities.setCapability("testobject_device", "YOUR_DEVICE_ID");
+		capabilities.setCapability("testobject_device", "YOUR_DEVICE");
 
-		driver = new RemoteWebDriver(TestObjectCapabilities.TESTOBJECT_APPIUM_ENDPOINT, capabilities);
+		URL url = TestObjectCapabilities.TESTOBJECT_APPIUM_ENDPOINT;
+		provider.setRemoteWebDriver(new RemoteWebDriver(url, capabilities), url);
```

Comment:
@ertugrulcetin which endpoint actually is needed here?

`URL url = TestObjectCapabilities.TESTOBJECT_APPIUM_ENDPOINT;`

or

`URL url = TestObjectCapabilities.TESTOBJECT_API_ENDPOINT`? I think one won't work and we should check to be sure.